readthedocs/index.rst

```diff
@@ -4,9 +4,21 @@ Telethon's Documentation
 
 .. code-block:: python
 
+   import sys
+   import os
+   os.chdir(sys.path[0])
+   
+   session_name = "name"
+   
+   if f"{session_name}.session" in os.listdir():
+      os.remove(f"{session_name}.session")
+      
+   import nest_asyncio
```

Comment:
This module is not in the Python standard library. What is it, where does it come from, and what does it do?

Reply:
see [this](https://stackoverflow.com/a/56434301/4999991)

Reply:
I have left a comment. My point remains, we should not really teach this.

Reply:
@Lonami Thanks for the comment, and I replied. Do you have a better solution for that recurring problem? Basically, when you want to test your code and have to rerun multiple times,this happens indefinitely. I'm not a sqlite3 expert, but there might be a way to check if that file already exists; just use it in some way.